feat(js): Add SvelteKit SDK Docs (#6808)
getsentry/sentry-docs: src/platform-includes/performance/enable-tracing/javascript.sveltekit.mdx

> The Sentry SvelteKit SDK comes with performance monitoring included. To enable it, you just need to configure your sample rate as detailed below.

Reviewer: ```suggestion
The Sentry SvelteKit SDK comes with performance monitoring included. To enable it, configure the sample rate as described below.
```
